libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -344,6 +390,8 @@ async def stream(
         async with self.client.stream(
             method, path, headers=request_headers, content=content, params=params
         ) as res:
+            if on_response:
+                on_response(res)
```

Comment:
should we say this is async? (would someone ever want to decode the response here?)